Packages/com.unity.inputsystem/Documentation~/HID.md

```diff
@@ -1,49 +1,51 @@
+Every HID comes with a device descriptor. To browse through the descriptor of an HID from the Input Debugger, click the __HID Descriptor__ button in the device debugger window. To specify the type of the device, the HID descriptor reports entry numbers in the [HID usage tables](https://www.usb.org/document-library/hid-usage-tables-112), and a list of all controls on the device, along with their data ranges and usages.
 
 ![HID Descriptor](Images/HIDDescriptor.png)
 
 The Input System handles HIDs in one of two ways:
 
 1. The system has a known layout for the specific HID.
-2. A layout is auto-generated for the HID on the fly.
+2. If the system does not have a known layout, it auto-generates one for the HID.
 
 ## Auto-generated layouts
 
 By default, the Input System creates layouts and Device representations for any HID which reports its usage as `GenericDesktop/Joystick`, `GenericDesktop/Gamepad`, or `GenericDesktop/MultiAxisController` (see the [HID usage table specifications](https://www.usb.org/document-library/hid-usage-tables-112) for more information). To change the list of supported usages, set [`HIDSupport.supportedHIDUsages`](../api/UnityEngine.InputSystem.HID.HIDSupport.html#UnityEngine_InputSystem_HID_HIDSupport_supportedHIDUsages).
 
 When the Input System automatically creates a layout for an HID, it always reports these Devices as [`Joysticks`](Joystick.md), represented by the [`Joystick` device class](../api/UnityEngine.InputSystem.Joystick.html). The first elements with a reported HID usage of `GenericDesktop/X` and `GenericDesktop/Y` together form the joystick's [`stick`](../api/UnityEngine.InputSystem.Joystick.html#UnityEngine_InputSystem_Joystick_stick) Control. The system then adds Controls for all further HID axis or button elements, using the Control names reported by the HID specification. The Input System assigns the first control with an HID usage of `Button/Button 1` to the joystick's [`trigger`](../api/UnityEngine.InputSystem.Joystick.html#UnityEngine_InputSystem_Joystick_trigger) Control.
 
-The auto-generated layouts represent a "best effort" on the part of the Input System. Because the way HIDs describe themselves as per standard is too ambiguous in practice, generated layouts might lead to Controls that don't work as expected. For example, while the layout builder can identify hat switches and D-pads, it can often only make guesses as to which direction represents which. The same goes for individual buttons, which generally aren't assigned any meaning in HID.
+The auto-generated layouts represent a "best effort" on the part of the Input System. The way standard HIDs describe themselves is too ambiguous, so generated layouts might lead to Controls that don't work as expected. For example, while the layout builder can identify hat switches and D-pads, it can often only make guesses as to which direction represents which. The same goes for individual buttons, which generally aren't assigned any meaning in HID.
```

Comment:
I think HIDs here refers to Human Interface Devices (the devices, not the standard). So technically, the meaning of the second sentence is, "The way Human Interface Devices describe themselves as per/in accordance with the HID standard is too ambiguous in practice", as in the standard-complying self-description isn't useable in a real-life scenario. We should keep that meaning.